src/containers/OwnerView.js

```diff
@@ -11,10 +11,14 @@ import {
   BottomAnchor,
   TopAnchor,
   Column,
+  Row,
+  Card,
 } from '../components/CustomStyled';
+import PartyChip from '../components/PartyChip';
 
 function OwnerView({ owner }) {
   const [characters, updateCharacters] = useState([]);
+  const [parties, updateParties] = useState({});
```

Comment:
why an object and not an array?

Reply:
I wanted the key to be the name of the party and have an array of character id's.

Reply:
My concern is allowing a user submission to be an object key seems like we're asking for trouble, it also seems a bit obfuscated since its more of a collection than an object. it may not be as pretty but i think just handling the indexes may be more direct. what do you think tho?

Reply:
Oh shoot, that's a good point i always forget about users doing obnoxious things like putting special characters and junk in there